lib/splitclient-rb/cache/repositories/flag_sets/redis_repository.rb

```diff
@@ -0,0 +1,40 @@
+require 'concurrent'
+
+module SplitIoClient
+  module Cache
+    module Repositories
+      class RedisFlagSetsRepository < Repository
+
+        def initialize(config)
+          super(config)
+          @adapter = SplitIoClient::Cache::Adapters::CacheAdapter.new(@config)
+        end
+
+        def flag_set_exist?(flag_set)
+          @adapter.exists?(namespace_key(".flagSet.#{flag_set}"))
+        end
+
+        def get_flag_set(flag_set)
+          @adapter.get_set(namespace_key(".flagSet.#{flag_set}"))
```

Comment:
are you using pipeline for this operation ???

Reply:
will use pipline